src/components/AddButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  good: string,
+  isSelected: boolean,
+  handleGoodToggle: (good: string, isSelected: boolean) => void,
+};
+
+export const AddButton: FC<Props> = ({
```

Comment:
I liked the idea of creating a different component for button and toggle logic. 
My suggestion would be changing the name of the component because it adds or removes goods.

Reply:
Totally agree 💯